plugin/mouse/mouse.py

```diff
@@ -143,6 +143,13 @@ def mouse_drag_end():
         for button in buttons_held_down:
             ctrl.mouse_click(button=button, up=True)
 
+    def toggle_mouse_drag(button: int):
```

Comment:
I would name this `mouse_drag_toggle` to be consistent with the other mouse actions